templates/telemetry_derived/devtools_events_amplitude_v1/view.sql

```diff
@@ -0,0 +1,523 @@
+        WHEN (event_category IN ('devtools.main') ) AND (event_method IN ('device_added') ) AND (event_object IN ('aboutdebugging') ) THEN 'dt_adbg - device_added' 
+        WHEN (event_category IN ('devtools.main') ) AND (event_method IN ('device_removed') ) AND (event_object IN ('aboutdebugging') ) THEN 'dt_adbg - device_removed' 
+        WHEN (event_category IN ('devtools.main') ) AND (event_method IN ('inspect') ) AND (event_object IN ('aboutdebugging') ) THEN 'dt_adbg - inspect' 
+        WHEN (event_category IN ('devtools.main') ) AND (event_method IN ('open_adbg') ) AND (event_object IN ('aboutdebugging') ) THEN 'dt_adbg - open_adbg' 
+        WHEN (event_category IN ('devtools.main') ) AND (event_method IN ('runtime_added') ) AND (event_object IN ('aboutdebugging') ) THEN 'dt_adbg - runtime_added' 
+        WHEN (event_category IN ('devtools.main') ) AND (event_method IN ('runtime_connected') ) AND (event_object IN ('aboutdebugging') ) THEN 'dt_adbg - runtime_connected' 
+        WHEN (event_category IN ('devtools.main') ) AND (event_method IN ('runtime_disconnected') ) AND (event_object IN ('aboutdebugging') ) THEN 'dt_adbg - runtime_disconnected' 
+        WHEN (event_category IN ('devtools.main') ) AND (event_method IN ('runtime_removed') ) AND (event_object IN ('aboutdebugging') ) THEN 'dt_adbg - runtime_removed' 
+        WHEN (event_category IN ('devtools.main') ) AND (event_method IN ('select_page') ) AND (event_object IN ('aboutdebugging') ) THEN 'dt_adbg - select_page' 
+        WHEN (event_category IN ('devtools.main') ) AND (event_method IN ('show_profiler') ) AND (event_object IN ('aboutdebugging') ) THEN 'dt_adbg - show_profiler' 
+        WHEN (event_category IN ('devtools.main') ) AND (event_method IN ('update_conn_prompt') ) AND (event_object IN ('aboutdebugging') ) THEN 'dt_adbg - update_conn_prompt'
+    END AS event_name,
+    event_timestamp AS timestamp,
+    (event_timestamp + created) AS time,
+    app_version,
+    os AS os_name,
+    os_version,
+    country,
+    city,
+    (SELECT
+      ARRAY_AGG(CONCAT('"',
+        CAST(key AS STRING), '":"',
+        CAST(value AS STRING), '"'))
+     FROM
+       UNNEST(event_map_values)) AS event_props_1,
+    event_map_values,
+    event_object,
+    event_value,
+    event_method,
+    event_category,
+    created,
+    NULL AS settings,
+    normalized_channel,
+    env_build_arch,
+    sample_id,
+    application_build_id,
+    app_name,
+    locale,
+    is_default_browser,
+    is_wow64,
+    memory_mb,
+    profile_creation_date,
+    attribution_source,
+    (SELECT ARRAY_AGG(CONCAT(key,'_',value.branch)) from UNNEST(experiments)) as experiments
+FROM
+    base_events
+WHERE 
+    (doc_type IN ('main', 'event') AND app_name = 'Firefox' AND normalized_channel IN ('nightly', 'beta', 'aurora'))
+    OR (doc_type = 'event' AND app_name = 'Firefox' AND normalized_channel = 'release' AND sample_id < 50)
+), all_events_with_insert_ids AS (
```

Comment:
Previously this was handled in two separate load jobs - one for prerelease events, another for 50% of release.